docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,70 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
-
+= Secure a Quarkus application with Basic authentication
 include::./attributes.adoc[]
+:categories: get-started
+:toc: preamble
```

Comment:
I hate to do this to you... but I think we've decided we can remove this (I'll handle it in the included attributes)

Reply:
Lines 8 and 9 removed.